Kata RPG/Kata RPG/Test.swift

```diff
@@ -7,9 +7,9 @@
 
 import Foundation
 
-let c1 = Character(name: "Link")
-let w1 = Warrior(name: "Ganon")
-let p1 = Priest(name: "Zelda")
+let Link = Warrior(name: "Link")
+let Ganon = Warrior(name: "Ganon")
+let Zelda = Priest(name: "Zelda")
```

Comment:
That's better, but if the reader does not know the game you're referring to, how does it help him read the tests ?